opal/mca/base/mca_base_var_enum.c

```diff
@@ -105,6 +109,126 @@ mca_base_var_enum_t mca_base_var_enum_bool = {
     .dump      = mca_base_var_enum_bool_dump
 };
 
+/* verbosity enumerator */
+static mca_base_var_enum_value_t verbose_values[] = {
+    {MCA_BASE_VERBOSE_NONE,  "none"},
+    {MCA_BASE_VERBOSE_ERROR, "error"},
+    {MCA_BASE_VERBOSE_WARN,  "warn"},
+    {MCA_BASE_VERBOSE_INFO,  "info"},
+    {MCA_BASE_VERBOSE_DEBUG, "debug"},
+    {MCA_BASE_VERBOSE_TRACE, "trace"},
+    {-1, NULL}
+};
+
+static int mca_base_var_enum_verbose_get_count (mca_base_var_enum_t *enumerator, int *count)
+{
+    *count = 5;
+    return OPAL_SUCCESS;
+}
+
+
+static int mca_base_var_enum_verbose_get_value (mca_base_var_enum_t *self, int index,
+                                                int *value, const char **string_value)
+{
+    if (0 > index || 4 < index) {
```

Comment:
Why are these limits hardcoded? I don't think they are even correct, are they? I see 6 valid values in verbose_values, not 5